ProjectFiles/Assets/Scripts/Game/Core/Parking/ServerParkingSpaceManager.cs

```diff
@@ -14,6 +15,13 @@ public class ServerParkingSpaceManager : ParkingSpaceManager {
         public event SpaceClaimedDelegate SpaceClaimedEvent;
 
 
+        public override void OnRoundEnd(ushort roundNumber) {
```

Comment:
might be being picky and is probably an edge case, but should we set a flag `OnRoundEnd` that says if a timer triggers while we are resetting the timers it shouldn't count?

Currently we could have:
1. End the round.
2. End timers 1,2,3,4,5,6
3. Timer 8 gets triggered
4. End timers 7,8,9

Would we want timer 8 to count?

Reply:
I vote no, because the flag will make no difference to the player.

Reply:
But wont it be the difference between the server saying they've claimed a space and not a space.

Could we then end up sending packets saying someones claimed a space in a round after the round has ended?

Reply:
I've changed this so that it doesn't happen anymore.

```cs
cancellingTimers = true;
foreach (var space in parkingSpacesBySpaceID.Values) {
    space.Timer.Reset();
}
cancellingTimers = false;
```

If a timer triggers and `cancellingTimers` is true then it is ignored.